example/build.yaml

```diff
@@ -25,3 +25,10 @@ builders:
     builder_factories: ["cssBuilder"]
     build_extensions: {"$package$": ["web/generated.css"]}
     auto_apply: root_package
+
+  movingBuilder:
```

Comment:
nit: the name of this builder is a bit misleading, it isn't moving files around. Maybe `fileDiagnosticsBuilder`? (same for the class etc).